applications/API/tests/test_forms.py

```
@@ -0,0 +1,288 @@
+        )
+        form = InsultReviewForm(data=form_data)
+
+        with self.assertRaises(ValidationError) as context:
+            form.clean()
+
+        expected_message = (
+            "Email Not Provided - You have selected that you wish to be contacted to know "
+            "the desired outcome of the review, but have not provided an email address. "
+            "Please change your results contact preference or enter a vaild email addrwss"
+        )
+        self.assertEqual(str(context.exception), expected_message)
```

Ruff (0.12.2)

Use a regular `assert` instead of unittest-style `assertEqual`

Replace `assertEqual(...)` with `assert ...`

(PT009)